build/istanbul-config.js

```diff
@@ -1,24 +1,20 @@
 'use strict';
 
-var path = require('path');
+var unitTestCoverageDirectory = './coverage/unit';
+var unitTestCoverageReportHtmlFile = './coverage/unit/index.html';
 
```

Comment:
I replaced the build directory with the ones generator by our generator, so I think this is more a question for you ;)